### ISSUE TITLE
[cocos2d-x] Cascading Color does not work - with suggested Fix

### ISSUE THREAD

**Issue by ForestRingGames** [EsotericSoftware/spine-runtimes#902]
Hey Mario and/or Nate,

Cascading Colors in Cocos2d-x does not work with SpineSkeletons. If cascadeColor is enabled and one sets the color of the parent of a Skeleton, the Skeleton keeps its color.

The reason is that the draw-function uses **color** instead of **displayedColor**, suggested fix:

```
void SkeletonRenderer::draw (Renderer* renderer, const Mat4& transform, uint32_t transformFlags) {
	SkeletonBatch* batch = SkeletonBatch::getInstance();

	Color3B nodeColor = getDisplayedColor(); // Changed from getColor();

        // .... rest of function
}
```

The same logic is applied in other Cocos2d-x nodes, so it looks like an oversight to me - or is there a reason not to do this?

**Comment by badlogic** [comment]
Definitely an oversight. I'll apply the fix to the 3.6-beta branch asap.

**Comment by badlogic** [comment]
Thanks for reporting, fixed in 3.6-beta branch!
